internal/config/config.go

```diff
@@ -38,6 +38,13 @@ func MustEnableUnitTestingMode() {
 }
 
 func EnableUnitTestingMode() error {
+	return EnableUnitTestingModeFull(true)
+}
+
+// Sets configuration options and logging to debug if not already set,
```

Comment:
"Sets configuration options and logging to debug"
The English here is ambiguous.